src/main/kotlin/screeps/api/Constants.kt

```diff
@@ -21,6 +21,8 @@ external interface ResourceConstant : StringConstant
 external interface MineralConstant : ResourceConstant
 external interface ColorConstant : IntegerConstant
 external interface DensityConstant : IntegerConstant
+external interface OrderConstant : StringConstant
+external interface MarketResourceConstant : ResourceConstant
```

Comment:
I think this should be the other way around: Any Resource can be traded on the market, however only ResourceConstant can be stored inside a Storage. 

```
external interface MarketResourceConstant : StringConstant
external interface ResourceConstant : MarketResourceConstant
````

Reply:
Maybe we also can find a better name for `MarketResourceConstant`. I'm thinking of `Tradeable`

Reply:
The reason I went for `MarketResourceConstant` was to conform with the other `*Constant` interface name.